attributes/default.rb

```diff
@@ -1,6 +1,7 @@
 default['packer']['url_base'] = 'https://dl.bintray.com/mitchellh/packer'
 default['packer']['version'] = '0.8.6'
 default['packer']['arch'] = kernel['machine'] =~ /x86_64/ ? 'amd64' : '386'
+default['packer']['zipfile'] = ''
```

Comment:
As you are setting the node.default for this inside the install_binary recipe is this required?